controllers/v1/users.js

```diff
@@ -84,7 +84,90 @@ module.exports = class Users {
 				return resolve(targetedSolutions)
 			} catch (error) {
 				return reject({
-					status: error.status || HTTP_STATUS_CODE['internal_server_error'].status,
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+
+					errorObject: error,
+				})
+			}
+		})
+	}
+
+	/**
+      * @api {post} /project/v1/users/programs?isAPrivateProgram=true&page=:page&limit=:limit&search=:search 
+      * Program List
+      * @apiVersion 1.0.0
+      * @apiGroup Users
+      * @apiHeader {String} X-auth-token Authenticity token
+      * @apiSampleRequest /project/v1/users/programs?isAPrivateProgram=true&page=:page&limit=:limit&search=:search 
+      * @apiUse successBody
+      * @apiUse errorBody
+      * @apiParamExample {json} Request:
+      *{
+            "state": "665d8df5c6892808846230e7",
+            "district": "668240135fb8bc3e93ceae39",
+            "block": "6682771aa845ef3e891db070",
+            "cluster": "668242835fb8bc3e93ceae44",
+            "role": "head_master,district_education_officer"
+        }
+       * @apiParamExample {json} Response:
+       * {
+       * "message": "Users programs fetched successfully",
+         "status": 200,
+         "result": {
+             "data": [
+                 {
+                     "_id": "5ff438b04698083dbfab7284",
+                     "externalId": "TEST_SCOPE_PROGRAM",
+                     "name": "TEST scope in program",
+                     "metaInformation" : {
+                        "state" : ["Karnataka"],
+                        "roles" : ["District Education Officer", "Head Master"]
```

Comment:
@Prajwal17Tunerlabs can we make it recommendedFor

Reply:
roles keyword replaced with recommendedFor.